src/components/molecules/SearchAutocomplete/inputWithTags.tsx

```diff
@@ -5,28 +5,21 @@ import Tag from "../../atoms/Tag";
 import * as S from "./styles";
 import { InputWithTagsProps } from "./interface";
 
-const InputWithTags: FC<InputWithTagsProps> = ({
-  text,
-  onRemoveTag,
-  setTag,
-}) => {
+const InputWithTags: FC<InputWithTagsProps> = ({ text, onRemoveTag }) => {
   const handleKeyDown = (e: React.KeyboardEvent<HTMLSpanElement>) => {
-    switch (e.key) {
-      case "Backspace":
-        break;
-      case "Enter":
-        e.preventDefault();
-        break;
-      default:
+    if (e.key === "Backspace") {
+      onRemoveTag();
+    } else if (e.key === "Enter") {
+      e.preventDefault();
```

Comment:
Oh I forgot to change it back to switch case, it was easier to debug this way.

Reply:
I will leave it this way, as I started to break one test case with this `switch case` "refactor" 🙈